recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+c_compiler_version:
+  - 15  # [arm64]
```

Comment:
```suggestion
  - 15                                 # [osx and arm64]
```

Reply:
can you add a comment as to what you saw without this? its somewhat strange that you got a terrible error without this.

Reply:
@hmaarrfk I think I'd have to temporarly remove the conda_build_config.yaml. I can't remember exactly what was the error message and the test logs got wiped away since it was more than one months ago. It could be related to 
```
 In file included from ./src/duktape/duktape.h:205:
  ./src/duktape/duk_config.h:459:7: error: 'TARGET_OS_BRIDGE' is not defined, evaluates to 0 [-Werror,-Wundef-prefix=TARGET_OS_]
```
but I'm not totally sure. In fact, perhaps this was an error related related to the Maxosx-SDK and not Clang itself. I've some flashback while browsing my search history. So perhaps the problem wasn't directly clang itself.

Reply:
We're gonna see what was the error message https://github.com/conda-forge/dukpy-feedstock/pull/20/commits/555a4a3256292d4473d9f0e5a6c3e52fdfb47e1b

Reply:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1117240&view=logs&j=7df89df7-19f1-5ccc-b210-32c1d05c0f56&t=c5e4c004-2534-57b9-b739-c5013de31afa&l=1288